Resources/Prototypes/_LostParadise/Roadmap/roadmap.yml

```
@@ -0,0 +1,27 @@
+  description: "Обновление Центкома!"
+  progress: 50
+  status: roadmap-goal-progress
+  order: 1
+
+- type: roadmap
+  id: "blusec"
+  name: "Обновление 2.2 - BLUSEC!"
+  description: "Перекрас СБ в новый модный цвет!"
+  progress: 37
+  status: roadmap-goal-progress
+  order: 2
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)